functions/exercise2/script.js

```diff
@@ -1 +1,30 @@
 "use strict";
+
```

Comment:
This code is too complicated. What you really want is:
- trim phrase
- if trimmed phrase is empty, then throw exception
- revert trimmed phrase (yes, you did something very similar in previous excercise)
- check if reverted trimmed phrase is equal to trimmed phrase - if so, then it is a palindrome
So please this algorithm as I implemented it in 6 lines (including 3 lines for checking if exception should be thrown and throwing it).